plugins/oauth2/class.oauth2.plugin.php

```diff
@@ -17,5 +17,6 @@ class OAuth2Plugin extends Gdn_OAuth2 {
     public function __construct() {
         $this->setProviderKey('oauth2');
         $this->settingsView = 'plugins/settings/oauth2';
+        $this->authenticationScheme = 'oauth2';
```

Comment:
Let's actually put the call to the setter here because, in the not too distant future we will want to separate the provider key and the authentication scheme alias for this plugin.